### PR TITLE
Belanosima: Version 2.000 added

### PR DESCRIPTION
**Pull request by yanone**
f9cfb5c: [gftools-packager] Belanosima: Version 2.000 added

* Belanosima Version 2.000 taken from the upstream repo https://github.com/docrepair-fonts/belanosima-fonts at commit https://github.com/docrepair-fonts/belanosima-fonts/commit/f7c622f51c1834290f8305ed1549c4f4fba8263c.

### CODE REVIEW

**Comment by m4rc1e** [comment]
@twardoch which family is this a fallback for? looks like Cooper Black to me.

**Comment by yanone** [comment]
Updated

## Belanosima: Version 2.000 added

---
 e7ac018: [gftools-packager] Belanosima: Version 2.000 added

* Belanosima Version 2.000 taken from the upstream repo https://github.com/docrepair-fonts/belanosima-fonts at commit https://github.com/docrepair-fonts/belanosima-fonts/commit/1c67420c6b44474624ddab9458c539a4ecefcd13.

File: ofl/belanosima/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: "Belanosima"
+designer: "The DocRepair Project, Santiago Orozco"
+license: "OFL"
+category: "SANS_SERIF"
+category: "DISPLAY"
```
Comment (RosaWagner) [inline ofl/belanosima/METADATA.pb:5]: category should be reversed since the main one should be listed second.

**Comment by yanone** [comment]
Updated

## Belanosima: Version 2.000 added

---
 0252a66: [gftools-packager] Belanosima: Version 2.000 added

* Belanosima Version 2.000 taken from the upstream repo https://github.com/docrepair-fonts/belanosima-fonts at commit https://github.com/docrepair-fonts/belanosima-fonts/commit/cea2e41b370f366a82c6a44981dfab9114a50471.

**Comment by yanone** [comment]
Updated

## Belanosima: Version 2.000 added

---
 6ce2242: [gftools-packager] Belanosima: Version 2.000 added

* Belanosima Version 2.000 taken from the upstream repo https://github.com/docrepair-fonts/belanosima-fonts at commit https://github.com/docrepair-fonts/belanosima-fonts/commit/9177d5cda68ac206b347f29af36e3cb14ff999d3.

**Comment by RosaWagner** [comment]
@yanone .notdef partialdiff and omega are emptys

**Comment by yanone** [comment]
Updated

## Belanosima: Version 2.000 added

---
 88dec8c: [gftools-packager] Belanosima: Version 2.000 added

* Belanosima Version 2.000 taken from the upstream repo https://github.com/docrepair-fonts/belanosima-fonts at commit https://github.com/docrepair-fonts/belanosima-fonts/commit/b772e9b7c44f32e17230f255fac9cac4bfaaa07f.

**Pull request merged**